packages/syft/src/syft/core/node/domain/client.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, client: Client) -> None:
     def requests(self) -> List[RequestMessage]:
 
         # relative
-        from ..common.node_service.get_all_requests.get_all_requests_messages import (
+        from ..common.node_service.object_request.object_request_messages import (
             GetAllRequestsMessage,
         )
 
```

Comment:
Q: In L83,84 why do we serialize and de serialize  the same message?
can't we send it directly?
<pre>
<code>
blob = serialize(msg, to_bytes=True)
msg = deserialize(blob, from_bytes=True)
</pre>
</code>

Reply:
Solved by 639ed60